pkg/volume/rbd/rbd_util.go

```diff
@@ -277,33 +283,34 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) (string, error) {
 			if err == nil {
 				break
 			}
-			glog.V(1).Infof("rbd: map error %v %s", err, string(output))
+			glog.V(1).Infof("rbd: map error %v, rbd output: %s", err, string(output))
 		}
 		if err != nil {
-			return "", fmt.Errorf("rbd: map failed %v %s", err, string(output))
+			return "", fmt.Errorf("rbd: map failed %v, rbd output: %s", err, string(output))
 		}
 		devicePath, found = waitForPath(b.Pool, b.Image, 10)
 		if !found {
-			return "", errors.New("Could not map image: Timeout after 10s")
+			return "", fmt.Errorf("Could not map Pool: %v image: %v Timeout after 10s", b.Pool, b.Image)
```

Comment:
Change `Pool: %v iamge: %v` to `image %s/%s:`.

We can simply use `<pool>/<image>` format in logs, which use less characters. See examples: https://github.com/krmayankk/kubernetes/blob/460d26bb934656a74098732a3f812d3ee0a40f1a/pkg/volume/rbd/rbd_util.go#L205

`<pool>/<image>` image format is also widely used in `rbd` command, e.g. `rbd map kube/testimage`.